services-js/commissions-search/src/App.js

```diff
@@ -50,13 +52,40 @@ class App extends React.Component {
     this.setState({ currentSeats: event.target.value });
   };
 
+  findCheckedBoxes4GA = event => {
+    var chk_arr = document.getElementsByClassName('cb');
```

Comment:
A few style things:

 - Use camelCase for JavaScript identifiers.
 - Don’t use `var`. Use `const` by default, `let` if you need to.